roles/core/log_client/molecule/default/verify.yml

```diff
@@ -0,0 +1,16 @@
+---
+- name: Verify
+  hosts: all
+  tasks:
+  - name: Check that files exists
+    stat:
+      path: "{{ item }}"
+    register: stat_results
+    with_items:
+      - /etc/rsyslog.conf
```

Comment:
`dest` is a variable in the task (even if `rsyslog_conf_path` is **always** /etc/rsyslog.conf in vars). If the value should change, this test will fail.
Moreover, there is no conditional to run this task, thus we don't need to verify the file exists if the converge did not fail. (the `template` module would fail if it could not write to `dest`)

Reply:
Removed and made the test smarter.

Reply:
fixed